component.go

```diff
@@ -339,3 +349,79 @@ func StopProc(c interface{}) bool {
 	vCom.FieldByName("Term").Close()
 	return true
 }
+
```

Comment:
The chunk below should be removed from `component.go`. Component discovery will be handled by the runtime using reflection package, hardcoding it here means zero flexibility of the package across applications. Websockets code should also be removed from this file.